lib/unit_tests_utils/postgresql_web_service_client.rb

```diff
@@ -1,9 +1,15 @@
 require 'httparty'
+require_relative 'postgresql_client'
 
 class UnitTestsUtils::PGWebServiceClient
-  attr_reader :manifest
+  attr_reader :manifest, :manifest_name_opt, :password_opt
```

Comment:
Can we drop the opt, and always sent `:manifest_name` and `:password` in the initialization (either with the given value or the default).

If so, I think we can decrease the number of `unless` and `if` spread through the functions, and keep it in this single place.

Reply:
Done